specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2023-09-01/QueryPackQueries.json

```diff
@@ -0,0 +1,616 @@
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The new or updated query contained within the Log Analytics QueryPack.",
+            "schema": {
+              "$ref": "#/definitions/LogAnalyticsQueryPackQuery"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "QueryPatch": {
+            "$ref": "./examples/QueryPackQueriesUpdate.json"
+          }
+        }
+      },
+      "delete": {
+        "description": "Deletes a specific Query defined within an Log Analytics QueryPack.",
+        "operationId": "Queries_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/QueryPackNameParameter"
+          },
+          {
+            "$ref": "#/parameters/QueryIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The query has been successfully removed from the Log Analytics QueryPack"
+          },
+          "204": {
+            "description": "The specified query or associated QueryPack does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "QueryDelete": {
+            "$ref": "./examples/QueryPackQueriesDelete.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AzureResourceProperties": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "format": "arm-id",
+          "description": "Azure resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Azure resource type"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true,
+          "description": "Read only system data"
+        }
```

Comment:
please use the proxy resource definition from common types for this.

Reply:
I meant just get rid of AzureResourceProperties and use the proxyresource definition instead

Reply:
Done.